website/content/docs/nia/enterprise/index.mdx

```diff
@@ -7,8 +7,21 @@ description: >-
 
 # Consul-Terraform-Sync Enterprise
 
-Consul-Terraform-Sync Enterprise features address organization complexities of collaboration, operations, scale, and governance. Please see the sidebar navigation on the left to view all enterprise features.
+Consul-Terraform-Sync (CTS) Enterprise is available with [Consul Enterprise](https://www.hashicorp.com/products/consul) and requires a Consul [license](/docs/nia/enterprise/license) to be applied.
 
-Enterprise features require a Consul [license](/docs/nia/enterprise/license) to be applied.
+Enterprise features of CTS address organization complexities of collaboration, operations, scale, and governance. CTS Enterprise supports an official integration with [Terraform Cloud](https://www.terraform.io/cloud) to extend insight into dynamic updates of your network infrastructure.
 
-These features are part of Consul-Terraform-Sync Enterprise which is part of [Consul Enterprise](https://www.hashicorp.com/products/consul).
+| Features | Open Source | Enterprise |
+|----------|-------------|------------|
+| | Self-Managed | Self-Managed |
+| Consul Namespace | Default namespace | Filter task triggers by namespace |
+| Network Automation | Terraform OSS | Terraform OSS or Terraform Cloud |
```

Comment:
It would be helpful to call out this row "Automation Engine" in place of "Network Automation" since it would be technically accurate and makes the solution more generic over specific for just network automation

Reply:
I'm supportive of updating the name, but wanted to make sure "Automation Engine" is a vocabulary we are committed to add for CTS since this will the first time it's used and introduced in any of the docs.

Another option could be "Automation Driver"

Reply:
I think Kim makes a good point here -- we've consistently referred to this as 'network automation', and I think changing the terminology at this point would be confusing.

Reply:
We have used "Automation Driver" in our literature before. I am open to that